### PR TITLE
Updating the Readme

### PR DESCRIPTION
**Pull request by danich1**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,6 +1,33 @@
 # Snorkeling
 
-A pilot project to extract medical indications (when a compound treats a disease) from the literature using [Snorkel](https://github.com/HazyResearch/snorkel).
+This repository stores data and code to scale up the extraction of biomedical relationships (i.e. Disease-Gene assocaitions, Compounds binding to Genes, Gene-Gene interactions etc.) from the Pubmed Abstracts.
```
Comment (ajlee21) [inline README.md:3]: typo

Comment (ajlee21) [reply README.md:3]: Maybe a bit nit picky, but what do you mean by "scale up" here?  My understanding was that the goal was to provide a way to automatically detect these types of relationships from abstracts.  Not sure what scaling up means in this context.

File: README.md [hunk 2]
```diff
@@ -1,6 +1,33 @@
...
+
+This work uses a subset of [Hetionet v1](https://doi.org/cdfk), which is a heterogenous network that contains pharmacological and biological information in the form of nodes and edges. 
+(Resource metagraph shown in figure below).
```
Comment (ajlee21) [inline README.md]: What do you mean by "resource metagraph"?  Do you mean just an example of a hetergeneous network?  Though it looks like you're highlighting certain paths here.  Might need to add a bit more detail on what the figure is showing.

File: README.md (hunk 1 above)
Comment (ajlee21) [reply README.md:3]: I think the second paragraph is starting to get at my confusion.  Maybe some restructuring of the content might help

File: README.md [hunk 3]
```diff
@@ -1,6 +1,33 @@
...
+This paradigm combines distant supervision with simple rules and heuristics written as labeling functions to automatically annotate large datasets.
+Unfortunately, it takes a significant amount of time and effort to write a useful label function.
+Because of this fact, we aimed to speed up this process by re-using label functions across edge types.
+Read the full paper [here](https://greenelab.github.io/text_mined_hetnet_manuscript/).
+
+![Highlighted edges used in Hetionet v1](https://raw.githubusercontent.com/greenelab/text_mined_hetnet_manuscript/3a040e78114208417d2b1784ae558fb323eabe01/content/images/figures/hetionet/metagraph_highlighted_edges.png "Metagraph of Hetionet v1")
+
+## Directories
+
+The main folders for this project are described below. For convention most of the folder names are based on the metagraph shown above.
```
Comment (ajlee21) [inline README.md]: typo

File: README.md [hunk 4]
```diff
@@ -1,6 +1,33 @@
...
+
+The main folders for this project are described below. For convention most of the folder names are based on the metagraph shown above. 
+
+| Name | Descirption |
+| ---- | ---- | 
+| [compound_disease](https://github.com/greenelab/snorkeling/tree/master/compound_disease) | Head folder that contains all relationships compounds and diseases may share |
+| [compound_gene](https://github.com/greenelab/snorkeling/tree/master/compound_gene) | Head folder that contains all relationships compounds and genes may share | 
+| [disease_gene](https://github.com/greenelab/snorkeling/tree/master/disease_gene) | Head folder that contains all realtionships disease and genes may share |
+| [gene_gene](https://github.com/greenelab/snorkeling/tree/master/gene_gene) | Head folder than contains all realtionships genes may share with each other |
+| [dependency cluster](https://github.com/greenelab/snorkeling/tree/master/dependency_cluster) | This folder contains a notebook for extracting and converting the results of another [study](https://zenodo.org/record/1495808#.XUmlR_wpBrk).
```
Comment (ajlee21) [inline README.md]: Not sure what this dependency cluster is based on the description.

**Review by ajlee21 — APPROVED** [review]
Really nice ReadMe!

Just some comments to try to clarify the wording a bit.

File: README.md (hunk 1 above)
Comment (danich1) [reply README.md:3]: That is the overall goal. For this project our focused shifted to seeing if we can speed up the annotation process by re-using label functions. This work is centered around that,

File: README.md (hunk 2 above)
Comment (danich1) [reply README.md]: resource metagraph -> database schema. It shows the overall content that is contained in the database. I updated the readme for clarity.

**Pull request merged**